package.json

```diff
@@ -32,6 +32,8 @@
   },
   "files": [
     "lib",
+    "docs",
```

Comment:
Why would an npm package include this directory?

Reply:
Some Plugins include this:
- eslint-plugin-jest (see [here](https://github.com/jest-community/eslint-plugin-jest/blob/master/package.json#L17-L20))
- typescript-eslint (see [here](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/package.json#L14-L20))

I think it is good to have the rules' documentation available.